package.json

```diff
@@ -75,6 +75,8 @@
     "babel-preset-stage-2": "^6.24.1",
```

Comment:
I assume you'll want to get the jest commands in here to replace the lerna run test commands?

Reply:
for some reason, typing `jest` in test, takes me to the `/jest` dir without running anything.  Maybe this is zsh being "helpful"...